drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -100,7 +107,8 @@ static inline int stm32_clock_control_on(const struct device *dev,
 	defined(CONFIG_SOC_SERIES_STM32F7X) || \
 	defined(CONFIG_SOC_SERIES_STM32F2X) || \
 	defined(CONFIG_SOC_SERIES_STM32WBX) || \
-	defined(CONFIG_SOC_SERIES_STM32G4X)
+	defined(CONFIG_SOC_SERIES_STM32G4X) || \
+	defined(CONFIG_SOC_SERIES_STM32WLX)
```

Comment:
could be sorted so that  CONFIG_SOC_SERIES_STM32WLX follows CONFIG_SOC_SERIES_STM32WBX (and not CONFIG_SOC_SERIES_STM32G4X)

Reply:
Done